### ISSUE TITLE
Herne Hill Market on front page

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#673]
New site.
We need to have a feature, somehow, of a box, probably on the right hand column that is about the market. It needs to have a picture in it and some text below. We need to be able to edit both. If a user clicks on the box then it takes them to the market area in the services/groups section.

Ideally we would be able to have more than one box so we could for a short period of time, feature everything to do with the Music festival, or similar. 

It gives a full time presence on the front page for an issue or area that we want to keep in front of people.

I suspect that VAC will want something like this as well.

Easy to do? can we rig it up on the c as a quickie?

**Comment by pmackay** [comment]
Can you just use the panels page editing options for that (the black bar at the bottom)?

**Comment by djwesto** [comment]
So what you're really asking for is a promotions box which allows you to add new panels promoting another page on the site?

You could do it straight away with a manually-created panel...

Have you tried to do this yourself yet?

**Comment by GilesGibson** [comment]
and do what? add a block? happy to try it, give us a clue.

**Comment by GilesGibson** [comment]
@djwesto yes, something like that. How do I add a panel and what should I avoid. Don't want to break the site....

**Comment by GilesGibson** [comment]
any clues as to how to do this?

**Comment by djwesto** [comment]
How's about I write a guide on how to do this?

**Comment by GilesGibson** [comment]
it the admin manual please.... ta everso!

**Comment by djwesto** [comment]
Hi Giles, do you have an image and some text you want to add to the sidebar panel?

**Comment by djwesto** [comment]
And which actual page on the live site are you wanting to link to?

**Comment by GilesGibson** [comment]
Idealy we could upload image and edit text to the block. Alos to be able to change the landing page it goes to. By letting the admin team administer it will reduce the burden on tech support longer term.

**Comment by pmackay** [comment]
@GilesGibson can you please provide a sample image and text you would like to put there, and what you would like them to link to?

**Comment by GilesGibson** [comment]
Link to the market section: http://www.hernehill.org.uk/groups/herne-hill-community-market

For now, if we can use the same image that is on the market page (above).

**Comment by pmackay** [comment]
@GilesGibson and what text would like in the homepage block?

**Comment by GilesGibson** [comment]
Sorry, forgot the text. 
"Herne Hill Community Market - all the information, events and details here"

**Comment by pmackay** [comment]
And when you’re logged in, are you able to go to http://www.hernehill.org.uk/admin/structure/block/add and create a new block with content?

**Comment by GilesGibson** [comment]
I get an error if I click on that link. If I go in via my login the structure menu does not have anything obvious about blocks listed.

**Comment by pmackay** [comment]
As a test I've popped a link to that Group in the sidebar on homepage by inserting it via the "Customise this page" button at bottom and then choosing "Content item" from the left sidebar. It needs some theme improvements though because the image is too small.

**Comment by GilesGibson** [comment]
I can click on "customise this page" and see the item. What I cannot figure out is the "left sidebar", haven't got this or it is called something else.

**Comment by pmackay** [comment]
Sorry, its on the right side (doh!) - I just mean the right hand column. The homepage has 3 columns in a 1-2-1 width layout ratio.

**Comment by GilesGibson** [comment]
OK, I had thought that what was you meant. I am not sure how to edit the content though, no options there give you access to the raw text/picture. I looked in "content" but could not find it either.

**Comment by pmackay** [comment]
Its inserted the content item, which means you cannot edit that directly. If you want more custom control, you can try the "Add text" link on the content sidebar. I've just done that as another example and custom created a title and inserted the image manually. You could then insert some custom text underneath. Better?

**Comment by GilesGibson** [comment]
OK, this is more what we were after, thanks. Couple of things:

How would we add a new block (History Hear project for instance) that operated in a similar fashion?

How would a site administrator edit the content of the box simply? Could it appear in the Content section and allow for a simple edit or would they have to go via "customize this page" which is a bit scary for some people.

**Comment by pmackay** [comment]
You could add "History Hear" using the same "Add text" option. 
There could be a content item that contained the text, but it would still need adding via the "customise the page" option. How often are you likely to need to edit the content of a block like that, once its been created?

**Comment by GilesGibson** [comment]
Ideally we would want to add another dedicated block for History Hear with its own picture and text. May need to edit it every few weeks and sometime remove from the front page.

**Comment by GilesGibson** [comment]
I have deleted the smaller test block, in case you were concerned. Hope this is OK.
